Code/Framework/AzToolsFramework/AzToolsFramework/ComponentModes/BoxViewportEdit.cpp

```diff
@@ -93,30 +84,45 @@ namespace AzToolsFramework
                 linearManipulator->SetViews(AZStd::move(views));
 
                 linearManipulator->InstallMouseMoveCallback(
-                    [this, entityComponentIdPair,
-                     transformScale{ linearManipulator->GetSpace().GetUniformScale() }](const LinearManipulator::Action& action)
+                    [this,
+                     entityComponentIdPair,
+                     transformScale{ linearManipulator->GetSpace().GetUniformScale() },
+                     nonUniformScale{ linearManipulator->GetNonUniformScale() }](const LinearManipulator::Action& action)
                 {
+                    const bool symmetrical = !m_allowAsymmetricalEditing || action.m_modifiers.Ctrl();
```

Comment:
suggestion: We might want to check with UX about which key to use for symmetric vs asymmetric editing (I'm fine with <kbd>Ctrl</kbd> but might be worth checking)

Reply:
Yeah, this was just a placeholder till I checked with UX